cylc/flow/cfgspec/workflow.py

```diff
@@ -1490,6 +1490,33 @@ def upg(cfg, descr):
         ['scheduling', 'hold after cycle point']
     )
 
+    u.deprecate(
+        '8.0.0',
+        ['runtime', '__MANY__', 'suite state polling'],
+        ['runtime', '__MANY__', 'workflow state polling'],
+    )
+
+    for setting in [
+        'user',
+        'host',
+        'interval',
+        'max-polls',
+        'message',
+        'run-dir',
+        'verbose mode'
+    ]:
+        u.deprecate(
+            '8.0.0',
+            ['runtime', '__MANY__', 'suite state polling', setting],
+            ['runtime', '__MANY__', 'workflow state polling', setting],
+        )
+
+    u.deprecate(
+        '8.0.0',
+        ['scheduler', 'simulation', 'disable suite event handlers'],
+        ['scheduler', 'simulation', 'disable workflow event handlers'],
+    )
```

Comment:
This ought to have be tested in `tests/functional/deprecations/01-cylc8-basic.t`

Reply:
Turns out this item was already obsoleted, so I've removed the deprecation, and added the obsolete item to the test.